### ISSUE TITLE
lock/unlock door

### ISSUE THREAD

**Issue by MZorzy** [msp1974/homeassistant-jlrincontrol#39]
hi,
on "lock.rrs_doors" (or lock.rrs_hst_doors) car i see all door status and position
but if try lock/unlock door have this error:

```
Logger: homeassistant.components.websocket_api.http.connection.1766230704
Source: custom_components/jlrincontrol/lock.py:62
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 11:22:09 (5 occurrences)
Last logged: 11:22:29
'JLRLock' object has no attribute 'config_entry'

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 130, in handle_call_service
    connection.context(msg),
  File "/usr/src/homeassistant/homeassistant/core.py", line 1260, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1295, in _execute_service
    await handler.func(service_call)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 209, in handle_service
    self._platforms.values(), func, call, required_features
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 454, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 597, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 485, in _handle_entity_call
    await result
  File "/config/custom_components/jlrincontrol/lock.py", line 62, in async_unlock
    jlr_service = JLRService(self._hass, self.config_entry, self._vin)
AttributeError: 'JLRLock' object has no attribute 'config_entry'
```

i have set PIN on config integrations Jaguar Landrover InControl options.
JLR 2.0.1

**Comment by msp1974** [comment]
Fixed in v2.0.2 along with some other minor issues.  Let me know.

**Comment by MZorzy** [comment]
yes. now work :+1:
